ts-tests/monday-sdk-js-module.test.ts

```diff
@@ -1,77 +1,125 @@
-import mondaySdk from '../types';
+import mondaySdk from "../types";
 const monday = mondaySdk();
 
-monday.api('test'); // $ExpectType Promise<{ data: object; }>
+monday.api("test"); // $ExpectType Promise<{ data: object; }>
 
-monday.setApiVersion('2023-10'); // $ExpectType void
-mondaySdk({ apiVersion: '2023-10' });
-monday.api('test', { apiVersion: '2023-07' }); // $ExpectType Promise<{ data: object; }>
+monday.setApiVersion("2023-10"); // $ExpectType void
+mondaySdk({ apiVersion: "2023-10" });
+monday.api("test", { apiVersion: "2023-07" }); // $ExpectType Promise<{ data: object; }>
 
-monday.setToken('test'); // $ExpectType void
+monday.setToken("test"); // $ExpectType void
 
-monday.get('context'); // $ExpectType Promise<any>
-monday.get('settings'); // $ExpectType Promise<any>
-monday.get('itemIds'); // $ExpectType Promise<any>
-monday.get('sessionToken'); // $ExpectType Promise<any>
+monday.get("context", { appFeatureType: "AppFeatureBoardView" }).then(res => {
+  const { data } = res;
+  data.app.id; // $ExpectType number
+  data.permissions.approvedScopes; // $ExpectType string[]
+  data.theme; // $ExpectType string
+  data.boardId; // $ExpectType number
+  data.viewMode; // $ExpectType string
```

Comment:
these assertions are not promising anything about the types of these primitives, i.e. if boardId changes to string, this test won't catch it. I recommend using https://www.npmjs.com/package/expect-type